fdbclient/ServerKnobs.cpp

```diff
@@ -581,6 +581,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 
 	init( MIN_AVAILABLE_SPACE,                                   1e8 );
 	init( MIN_AVAILABLE_SPACE_RATIO,                            0.05 );
+	init( MIN_AVAILABLE_SPACE_RATIO_SAFETY_BUFFER,              0.01 );
```

Comment:
I'm investigating a failure in AtomicBackupCorrectness when this is set to 0.03. One of the tests there seems to hit a condition where a server is very low on space and doesn't like it when it doesn't get a viable team result.